packages/webview_flutter/webview_flutter_android/test/legacy/surface_android_test.dart

```diff
@@ -116,3 +122,9 @@ class TestWebViewPlatformCallbacksHandler
   @override
   void onWebResourceError(WebResourceError error) {}
 }
+
+/// This allows a value of type T or T? to be treated as a value of type T?.
+///
+/// We use this so that APIs that have become non-nullable can still be used
+/// with `!` and `?` on the stable branch.
+T? _ambiguate<T>(T? value) => value;
```

Comment:
`_ambiguate` is back!

Reply:
yeah, and with a vengeance. :-/
it'll disappear again after the next stable, if this all goes according to plan.